### ISSUE TITLE
Learning Rate

### ISSUE THREAD

**Issue by tom99763** [yl-1993/Matrix-Capsules-EM-PyTorch#16]
https://www.kaggle.com/code/tom99763/matrix-capsules-with-em-routing?scriptVersionId=124263074

我發現使用0.001的learning rate沒辦法訓練這個模型(mnist的準確度卡在0.1), 我想知道為什麼訓練這篇paper的模型learning初始要大到0.01, 是gradient的問題嗎?

還有我有發現kernel_tile的運算產生的gradient非常大, 或許是這個問題?
